plugins/GSdx/GSdx.cpp

```diff
@@ -276,9 +276,10 @@ void GSdxApp::Init()
 	m_gs_offset_hack.push_back(GSSetting(2,  "Special", "Texture"));
 	m_gs_offset_hack.push_back(GSSetting(3,  "Special", "Texture - aggressive"));
 
+	m_gs_hw_mipmapping.push_back(GSSetting(-1, "Automatic", "Default"));
 	m_gs_hw_mipmapping.push_back(GSSetting(0, "Off", ""));
 	m_gs_hw_mipmapping.push_back(GSSetting(1, "Basic", "Fast"));
-	m_gs_hw_mipmapping.push_back(GSSetting(2, "Full", "Very Slow - Not Recommended"));
+	m_gs_hw_mipmapping.push_back(GSSetting(2, "Full", "Very Slow"));
```

Comment:
I think it's better to leave the original line and just change it to `Full (Slow - Not Recommended)`